README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# CSS Homework1
+
+> ## Fernando Alonso Regalado
+
+![Bio Picture](https://github.com/fernandoregalado/hw01/blob/master/picture_bio.jpg)
+
+Hi! I am a second year **MPP student** :neckbeard: and peruvian **economic analyst** :bowtie:.
+
+* :school_satchel: Education: [*Harris School of Public Policy*](https://harris.uchicago.edu/)
+* :bank: Work: [*Central Reserve Bank of Peru*](https://www.bcrp.gob.pe/en)
+* Contact information:
+  * :id: @fernandoregalado
```

Comment:
I think you did well in making your id, email, and phone number a new unordered list, which makes the file looks more organized.

Reply:
I agree, this looked really cool!